eng/pipelines/templates/steps/analyze.yml

```diff
@@ -3,15 +3,17 @@ parameters:
   ServiceDirectory: ''
   TestMarkArgument: ''
   AdditionalTestArgs: ''
+  Artifacts: []
 
 steps:
   - template: /eng/pipelines/templates/steps/analyze_dependency.yml
 
-  - task: PythonScript@0
-    displayName: 'Verify Change Log'
-    inputs:
-     scriptPath: 'scripts/devops_tasks/verify_change_log.py'
-     arguments: '"${{ parameters.BuildTargetingString }}" --service=${{parameters.ServiceDirectory}}'
+  - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
If we do end up needing to opt-out for specific packages we should do it by adding a setting to the artifact in the ci.yml file. Similar to what we do in the release pipeline at https://github.com/Azure/azure-sdk-for-python/blob/master/eng/pipelines/templates/stages/archetype-python-release.yml#L57.